src/test/operators/table_scan_like_test.cpp

```diff
@@ -40,18 +40,27 @@ class OperatorsTableScanLikeTest : public BaseTest {
     _gt_special_chars->execute();
 
     // load and compress string table
-    auto test_table_string_dict = load_table("src/test/tables/int_string_like.tbl", 5);
-    ChunkEncoder::encode_chunks(test_table_string_dict, {ChunkID{0}});
+    if (::testing::UnitTest::GetInstance()->current_test_info()->value_param()) {
```

Comment:
Yes - googletest doesn't let you call `GetParam()` on a non-parameterized test.

Reply:
Don't get it - and can't build your branch right now because of libpq to figure it out myself - you **are** parameterizing the Test here, aren't you `INSTANTIATE_TEST_CASE_P(EncodingTypes, OperatorsTableScanLikeTest`
When would `value_param()` return nullptr?

(by "optional" I meant `std::optional`, btw.)

Reply:
Some tests are parameterized `TEST_P` (those running on compressed columns), others not `TEST_F` (those running on ValueColumns).

Reply:
Should be two test cases then (one for high level functionality and one specifically for storage related optimizations) - but not your job here? 

Please at least leave a comment above this line as to why you're doing this.

Reply:
That's not necessarily for any optimizations. Some, like Run-Length are not optimized. In that way, they are all high-level.

I started putting those into two test cases, but found that they would basically duplicate most of the setup.

Reply:
alright, leave it as one test case, it's not like these are beautiful anyway. just the comment pls

Reply:
...which I would have put *above* the if, but that's up to you.